src/Diagnostics.RuntimeHost/Services/CacheService/TableCacheService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using Diagnostics.RuntimeHost.Services.CacheService.Interfaces;
+using System.Collections.Concurrent;
+using Diagnostics.ModelsAndUtils.Models;
+using Diagnostics.RuntimeHost.Services.StorageService;
+using System.Threading.Tasks;
+using Diagnostics.ModelsAndUtils.Models.Storage;
+using Diagnostics.RuntimeHost.Models;
+using System.Linq;
+using System;
+
+namespace Diagnostics.RuntimeHost.Services.CacheService
+{
+    public class TableCacheService : ITableCacheService
```

Comment:
nit: Looks like this Table Cache service is for DiagEntitites only. Should we rename this service class to be a very specific instead of generic name. (prob something like DiagEntityTableCacheService)